test/spec/ol/view.test.js

```diff
@@ -613,7 +642,7 @@ describe('ol.View', function() {
       expect(view.getHints()[ol.ViewHint.ANIMATING]).to.be(2);
 
       view.animate({
-        rotate: Math.PI,
+        rotation: Math.PI,
```

Comment:
This (and same below) was a typo in the test.  We were creating no-op animations (`rotate`) is not an animation property and asserting that the view was actually animating.  With the new no-op checking, these tests failed.  With the proper spelling of `rotation`, the tests pass again.